### PR TITLE
Added documentation for the JPEG generator issue

### PR DESCRIPTION
**Pull request by kaushikvelusamy**

### CODE REVIEW

**Review by zhenghh04 — COMMENTED** [review]
@kaushikvelusamy , maybe create a single page for JPEG, and then link that in the following page: docs/source/knownissues.rst

File: docs/source/config.rst [hunk 1]
```diff
@@ -172,6 +172,11 @@ dataset
 .. note :: 
   The DALI data loader configuration needs to be coupled with pytorch framework otherwise will fail.
 
+.. attention::
+  
+  For `format: jpeg`, it is not recommended to generate data due to its lossy compression nature. Instead, provide the path to original dataset in the `data_folder` parameter. 
+  More information on JPEG image generator is provided at :ref:`jpeg_generator_issue` section.
```
Comment (hariharan-devarajan) [inline docs/source/config.rst:178]: Could u describe a bit here how the data folder needs to be structured. Add an examples with directory hierarchy

```
data_dir
├── train
│   ├── XXX.JPEG
│   ├── XXX.JPEG
├── val
│   ├── XXX.JPEG
│   ├── XXX.JPEG


```

File: docs/source/knownissues.rst [hunk 2]
```diff
@@ -12,3 +12,6 @@ Limitations and future works
   - For npz, png, jpeg, we currently only support one sample per file case. Multiple samples per file case will be supported in future. We have limited support for hdf5 format for multiple samples per file cases. 
 
 * Profiler support: Darshan is only supported in LINUX system, and might not work well within container. 
+
+* JPEG image generator : It is not recommended to generate `format: jpeg` data due to its lossy compression nature. Instead, provide the path to original dataset in the `data_folder` parameter. More information at :ref:`jpeg_generator_issue` section.
```
Comment (hariharan-devarajan) [inline docs/source/knownissues.rst:16]: Again add the required directiry streucture for dlio benchmark.

**Review by hariharan-devarajan — CHANGES_REQUESTED** [review]
Mostly it looks good to me. Can u do these minor changes?

File: docs/source/knownissues.rst (hunk 2 above)
Comment (kaushikvelusamy) [reply docs/source/knownissues.rst:16]: Done.

File: docs/source/config.rst (hunk 1 above)
Comment (hariharan-devarajan) [reply docs/source/config.rst:178]: Can u point to that part directly here using a relative link. U can create a label and a link to it in rst.

File: docs/source/knownissues.rst (hunk 2 above)
Comment (kaushikvelusamy) [reply docs/source/knownissues.rst:16]: Done. I have added the directory structure label in the attention section.

**Review by hariharan-devarajan — APPROVED** [review]

**Pull request merged**